AGSnapshotHelperTests/Categories/NSString+ContentSizeCategoryTests.m

```diff
@@ -0,0 +1,60 @@
+//
+//  NSString+ContentSizeCategoryTests.m
+//  AGSnapshotHelper
+//
+//  Created by Adam Grzegorowski on 22/11/15.
+//  Copyright (c) 2015 allegro. All rights reserved.
+//
+
+#import "NSString+ContentSizeCategory.h"
+
+#import <XCTest/XCTest.h>
+#import <UIKit/UIKit.h>
+
+@interface NSString_ContentSizeCategoryTests : XCTestCase
+
+@end
+
+@implementation NSString_ContentSizeCategoryTests
+
+- (void)testCategoryShouldReturnYesForValidStringOfContentSizeCategory {
+
+    // Arrange
+    NSArray *contentSizeCategories = @[
+        UIContentSizeCategoryExtraSmall,
+        UIContentSizeCategorySmall,
+        UIContentSizeCategoryMedium,
+        UIContentSizeCategoryLarge,
+        UIContentSizeCategoryExtraLarge,
+        UIContentSizeCategoryExtraExtraLarge,
+        UIContentSizeCategoryExtraExtraExtraLarge,
+        UIContentSizeCategoryAccessibilityMedium,
```

Comment:
I am wondering whether UIContentSizeCategoryAccessibility constants do not need separate test. Up to you.